eZ/Publish/Core/REST/Server/Controller/Location.php

```diff
@@ -46,18 +47,31 @@ class Location extends RestController
      */
     protected $trashService;
 
+    /**
+     * URLAlias Service.
+     *
+     * @var \eZ\Publish\Core\Repository\URLAliasService
```

Comment:
Please type-hint against API (kinda related to the service you've injected above ;))